.docksal/docksal.yml

```diff
@@ -1,6 +1,8 @@
 version: "2.1"
 
 services:
+  db:
+    hostname: database
```

Comment:
Does Lando use "database" as the hostname and is that why you are changing it here?

Reply:
Yes. It's easier to change in docksal than it is in lando.